packages/libpluto/build.sh

```diff
@@ -0,0 +1,48 @@
+TERMUX_PKG_HOMEPAGE=https://plutolang.github.io/
+TERMUX_PKG_DESCRIPTION="Shared library for the Pluto interpreter"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="Komo @cattokomo"
+TERMUX_PKG_VERSION="0.9.4"
+TERMUX_PKG_SRCURL=https://github.com/PlutoLang/Pluto/archive/refs/tags/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=6a4fae052a9ad47e29a6767047c3e55ab5c887486d14e39248657ff43b45875c
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="libc++"
+TERMUX_PKG_BUILD_DEPENDS="readline"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_BREAKS="plutolang (<< 0.9.4-2)"
+TERMUX_PKG_REPLACES="plutolang (<< 0.9.4-2)"
+
+# can't use php version from termux repo, there's no build for the latest version on the pre-built binaries
+_php_version="8.3.9"
```

Comment:
These two can just be `local` variables int the `termux_step_configure()` function.